webxr/openxr/input.rs

```diff
@@ -267,6 +357,75 @@ impl OpenXRInput {
             let binding_squeeze = Binding::new(&self.action_squeeze, path_squeeze);
             ret.push(binding_squeeze);
         }
+        self.action_buttons_common
+            .iter()
+            .enumerate()
+            .for_each(|(i, action)| {
+                let button_path = interaction_profile.standard_buttons[i];
+                if button_path != "" {
+                    let path = instance
+                        .string_to_path(&format!("/user/hand/{}/input/{}", hand, button_path))
+                        .expect(&format!(
+                            "Failed to create path for /user/hand/{}/input/{}",
+                            hand, button_path
+                        ));
+                    let binding = Binding::new(action, path);
+                    ret.push(binding);
+                }
+            });
+
+        if !interaction_profile.left_buttons.is_empty() && hand == "left" {
```

Comment:
There looks like a bunch of repetition in this code. Would it be possible to extract a helper that takes a slice of buttons/axes/etc, the corresponding slice from the interaction profile, and the hand, and returns the new bindings?

Reply:
Yes, I'll set that up